components/new-pedal-form/index.js

```diff
@@ -65,11 +64,11 @@ export default function NewPedalForm() {
     console.log(newPedal);
```

Comment:
I was a bit confused that your new entries are not displayed - and then was glad that you have this console.log. Still, we want to avoid merging log messages into our main.

Reply:
now new pedal is shown in pedal gallery on home